cluster/addons/addon-manager/kube-addons.sh

```diff
@@ -226,10 +228,10 @@ log INFO "== Default service account in the ${SYSTEM_NAMESPACE} namespace has to
 # Create admission_control objects if defined before any other addon services. If the limits
 # are defined in a namespace other than default, we should still create the limits for the
 # default namespace.
-for obj in $(find /etc/kubernetes/admission-controls \( -name \*.yaml -o -name \*.json \)); do
+while IFS=$'\n' read -r obj; do
   start_addon "${obj}" 100 10 default &
   log INFO "++ obj ${obj} is created ++"
-done
+done < <(find /etc/kubernetes/admission-controls \( -name \*.yaml -o -name \*.json \))
```

Comment:
change reason:
```
In ./cluster/addons/addon-manager/kube-addons.sh line 229:
for obj in $(find /etc/kubernetes/admission-controls \( -name \*.yaml -o -name \*.json \)); do
           ^-- SC2044: For loops over find output are fragile. Use find -exec or a while read loop.
```

Reply:
Is this better?  If that find returns a file with a space in it?...